libtrip-notification/src/main/res/values-el/strings.xml

```diff
@@ -1,19 +1,15 @@
 <resources>
     <!-- Notification Strings -->
-    <string name="end_navigation">Τέλος πλοήγησης</string>
-    <string name="kilometers">χλμ</string>
-    <string name="meters">μ</string>
-    <string name="miles">μίλια</string>
-    <string name="feet">πόδια</string>
```

Comment:
unused resources, it's duplicated with `libnavigation-util` resources

Reply:
Great catch! That was stale from the previous refactor 👍

Reply:
```xml
<string name="kilometers">χλμ.</string>
<string name="meters">μέτρ.</string>
<string name="miles">μίλ.</string>
<string name="feet">πόδ.</string>
```

`libnavigation-utils` here are slightly different though (they seem abbreviated 👀 `.` at the end). In any case, whatever comes from Transifex is the source of truth so we are good here 🚀

Reply:
right, I see the difference too. I haven't check which one is correct, but I removed this one because it's not in the right module too. Let me check Transifex which one is correct.

Reply:
<img width="506" alt="Screen Shot 2020-08-17 at 08 58 55" src="https://user-images.githubusercontent.com/3918950/90416876-e36c4b80-e067-11ea-8130-fedb1ea5be43.png">

this is what we have on Transifex, we are good here.

Reply:
You don't need to 👍 

> In any case, whatever comes from Transifex is the source of truth so we are good here 🚀

Whenever we pull the latest Transifex translations will be updated 💃

Reply:
do we have corporative account at `transifex`?